README.Rmd

```diff
@@ -143,9 +143,9 @@ adsl %>%
 
 The `xportr_metadata()` function can reduce duplication by setting the variable specification and domain explicitly at the top of a pipeline. If you would like to use the `verbose` argument, you will need to set in each function call.
 
-```{r, message=FALSE, eval=FALSE}
```

Comment:
Should we add a new chunk after this one to show off the new `xportr()` function? (show the alternative of the all-in-one)

```R
xportr(
  adsl,
  var_metadata = var_spec,
  df_metadata = dataset_spec,
  domain = "ADSL",
  path = "adsl.xpt"
)
````